backends/tfhe-cuda-backend/implementation/include/integer.h

```diff
@@ -351,6 +354,58 @@ template <typename Torus> struct int_radix_lut {
     }
   }
 
+  // constructor to reuse memory
+  int_radix_lut(cuda_stream_t *stream, int_radix_params params,
```

Comment:
@bbarbakadze you create the constructor but I don't see where it's used in the changes?